src/app/beer_garden/db/mongo/models.py

```diff
@@ -445,7 +445,7 @@ def save(self, *args, **kwargs):
             output_json = json.dumps(self.output)
             if len(output_json) > REQUEST_MAX_PARAM_SIZE:
                 self.logger.info("Output size too big, storing in gridfs")
-                self.output_gridfs.put(output_json, encoding=encoding)
+                self.output_gridfs.put(self.output, encoding=encoding)
```

Comment:
This is kind of a separate gridfs output issue that happened to be discovered while troubleshooting the original issue.  We made a change in 3.7.1 to how we decide when to store in gridfs.  As part of that change, we switched from storing the raw output to storing the serialized output.  I don't **think** that was intentional, but if it was then that was likely poor direction on my part.  I believe the intent was to serialize the output as a means of roughly estimating the storage size, but not to alter how the output is actually stored.  Storing the serialized version of the output breaks both the UI rendering and presumably any code users have written against request output that was expecting it in the old format.  This reverts the actual `put` line back to what it was in [3.6.0](https://github.com/beer-garden/beer-garden/blob/3.6.0/src/app/beer_garden/db/mongo/models.py#L436).